addons/mail/static/src/core/web/messaging_menu.js

```diff
@@ -100,7 +100,8 @@ export class MessagingMenu extends Component {
             partner: this.store.odoobot,
             isShown:
                 this.store.discuss.activeTab === "main" &&
-                this.notification.permission === "prompt",
+                this.notification.permission === "prompt" &&
+                !isDisplayStandalone(),
```

Comment:
This works in chrome linux in the PWA, maybe you need to target iOS specifically?

Reply:
In chrome linux PWA, push notification permissions is different from PWA?
If that's the case, then that's probably an iOS-specific issue

Reply:
They are the same permission but the bug you describe doesn't exist.

Reply:
Ok, I think I'll just remove this "fix" and just focus on bottom spacing, which is the core of the PR

Reply:
I removed this part of PR, so now it only focuses on bottom spacing.